opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -2412,6 +2412,160 @@ mod tests {
             empty_attrs_data_point.value, 6,
             "Empty attributes value should be 3+3=6"
         );
+
+        // Phase 2 - for delta temporality, after each collect, data points are cleared
```

Comment:
Is this test (phase 2) related to cardinality limit?

Reply:
yes. it is validating that cardinality limit is per cycle for delta.

Reply:
ok please ignore. Seems its checking for overflow.